src/app/services/content-management.service.ts

```diff
@@ -1179,12 +1181,21 @@ export class ContentManagementService {
   }
 
   unlockNode(node: NodeEntry) {
-    this.contentApi.unlockNode(node.entry.id).catch(() => {
-      this.store.dispatch(
-        new SnackbarErrorAction('APP.MESSAGES.ERRORS.UNLOCK_NODE', {
-          fileName: node.entry.name
-        })
-      );
-    });
+    this.contentApi.unlockNode(node.entry.id).subscribe(
+      nodeResult => {
+        delete node.entry.properties['cm:lockType'];
+        this.store.dispatch(new SetSelectedNodesAction([node]));
+        if (isPreview) {
```

Comment:
don't think this works as expected since it's a function https://github.com/Alfresco/alfresco-content-app/blob/development/projects/aca-shared/rules/src/navigation.rules.ts#L32